### PR TITLE
recipe for Pyrad MCH

### PR DESCRIPTION
**Pull request by meteoswiss-mdr**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by meteoswiss-mdr** [comment]
Looks like all tests were passed. Is there anybody that can help now?

**Comment by meteoswiss-mdr** [comment]
@conda-forge/staged-recipes. All tests have passed. It looks like the recipe is ready to go.

File: recipes/pyrad_mch/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    - "{{ PYTHON }} -m pip install . --no-deps -vvv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+  host:
+    - python
+    - pip
+    - pyart_mch
```
Comment (scopatz) [inline recipes/pyrad_mch/meta.yaml]: Is this needed to install?

Comment (meteoswiss-mdr) [reply recipes/pyrad_mch/meta.yaml]: Pyrad is basically a wrapper to pyart.

Comment (scopatz) [reply recipes/pyrad_mch/meta.yaml]: sure, but do you need it to build the actual package, or is it only a runtime dependency?

**Comment by meteoswiss-mdr** [comment]
@scopatz ,
You were right. It was only a runtime dependency. I removed it from the host in the recipe.

**Comment by scopatz** [comment]
Great!

**Pull request merged**